tests/sessions/python_autokitteh_funcs.txtar

```diff
@@ -0,0 +1,10 @@
+-- main.py main --
+
+from autokitteh import nop
+
+nop()
+
+def main(_):
+    nop()
```

Comment:
```suggestion
from autokitteh.connections import check_connection_name

check_connection_name("Grumpy")

def main(_):
    check_connection_name("Garfield")
```

Reply:
`check_connection_name` is an internal sdk function, but i guess i can access it somehow?

Reply:
done